audio_analysis.go

```diff
@@ -0,0 +1,119 @@
+package spotify
+
+import (
+	"encoding/json"
+	"fmt"
+	"net/http"
+)
+
+// AudioAnalysis contains a detailed audio analysis for a single track identified by its unique Spotify ID.
+// See https://developer.spotify.com/web-api/get-audio-analysis/
+//
+// Spotify's documentation is currently missing the object model for the AudioAnalysis.
+// See https://github.com/spotify/web-api/issues/317
+//
+// Also see The Echo Nest documentation
+// https://web.archive.org/web/20160528174915/http://developer.echonest.com/docs/v4/_static/AnalyzeDocumentation.pdf
+type AudioAnalysis struct {
+	Bars     []Measure          `json:"bars"`
+	Beats    []Measure          `json:"beats"`
+	Meta     AudioAnalysisMeta  `json:"meta"`
+	Sections []Section          `json:"sections"`
+	Segments []Segment          `json:"segments"`
+	Tatums   []Measure          `json:"tatums"`
+	Track    AudioAnalysisTrack `json:"track"`
+}
+
+// Measure represents beats, bars, tatums and are used in segments and sections descriptions.
+type Measure struct {
+	Start      float64 `json:"start"`
+	Duration   float64 `json:"duration"`
+	Confidence float64 `json:"confidence"`
+}
+
+type AudioAnalysisMeta struct {
```

Comment:
Can you add doc comments for `AudioAnalysisMeta`, `Section`, `Segment`, and `AudioAnalysisTrack`?